llvm/docs/AMDGPUUsage.rst

```diff
@@ -1016,6 +1016,15 @@ supported for the ``amdgcn`` target.
   `ptr addrspace(7)` directly, which produces a buffer fat pointer with an initial
   offset of 0 and prevents the address space cast from being rewritten away.
 
+  The `align` attribute on operations from buffer fat pointers is deemed to apply
```

Comment:
IIUC, we need to use double ` here.

Reply:
from the web page, single looks like a italic and mono, but double looks like a normal mono. I think the italic version looks really weird.

the wording looks good to me.